tools/library/src/reference/gemm_fp_other.cu

```diff
@@ -77,6 +77,24 @@ void initialize_gemm_reference_operations_fp_other(Manifest &manifest) {
     complex<double>,
     complex<double>
   >(manifest);
+
```

Comment:
these should not put in `gemm_fp_mixed_input.cu`?

Reply:
Yes, I added the mixed input gemm reference in a new file and forgot to remove it from here during the dev. Thanks for catching it. Resolved!